### ISSUE TITLE
Breaks Find Symbol in Editor

### ISSUE THREAD

**Issue by markaconrad** [yzhang-gh/vscode-markdown#1320]
Type: <b>Bug</b>

With this plugin enabled, cmd+shift+o (Find symbol in editor) just spins and never shows me symbols.

Extension version: 3.5.1
VSCodium version: VSCodium 1.81.0 (ee0ba35e0f5ffb18669618446b41de0e99569167, 2023-08-04T12:07:39.095Z)
OS version: Darwin arm64 22.6.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Pro (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|19, 11, 9|
|Memory (System)|16.00GB (0.30GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>

**Comment by yzhang-gh** [comment]
Sounds strange. You can use VS Code's troubleshooting to help find the reason. Please find more in https://code.visualstudio.com/docs/getstarted/keybindings#_troubleshooting-keybindings

**Comment by mstaal** [comment]
I agree with @markaconrad . I just came to the same conclusion using VSCodes Bisect functionality.

**Comment by yzhang-gh** [comment]
With the link I provided above, you can clearly see what happens inside VS Code after you press any key.

![image](https://github.com/yzhang-gh/vscode-markdown/assets/7588612/6220cc6c-61fb-4448-a0bb-ed745f8d3cf1)

**Comment by hughfenghen** [comment]
Pressing `cmd - t` is expected to search symbols in the workspace.

If the plugin is enabled, only the titles of markdown are displayed. When the plugin is disabled, symbols are displayed correctly


**Actual**
![image](https://github.com/yzhang-gh/vscode-markdown/assets/3307051/949c9298-54e7-4a64-afd3-ad057216ad82)

**Expect**
![image](https://github.com/yzhang-gh/vscode-markdown/assets/3307051/e3732e22-c6e8-4a61-ba1a-a79b16d19fe7)
